README.md

```diff
@@ -5,46 +5,196 @@
 ![Azure DevOps builds (branch)](https://img.shields.io/azure-devops/build/openvinoci/b2bab62f-ab2f-4871-a538-86ea1be7d20f/13?label=Public%20CI)
 [![PyPI Downloads](https://pepy.tech/badge/openvino)](https://pepy.tech/project/openvino)
```

Comment:
As a developer I would love to see more badges here describing the current state of the repo. For instance, python versions on which `openvino` runs. The user would visually want to see this in a badge. Something like the following
![image](https://user-images.githubusercontent.com/17574420/160754894-44b89bfb-2df5-4daa-b63e-ccea452545bb.png)

In addition, if possible, it would be nice to show the coverage badge as well. All these extra badges would show how rigorously openvino is tested.
![image](https://user-images.githubusercontent.com/17574420/160755135-2d45b415-6bb6-4b3b-a6f0-9ee0e4103b8f.png)

Reply:
We have a ticket to add code coverage in public CI